kge/model/kge_model.py

```diff
@@ -488,24 +489,25 @@ def create(
             else:
                 model_name = config.get("model")
             class_name = config.get(model_name + ".class_name")
-            module = importlib.import_module("kge.model")
+
+            modules = [importlib.import_module(m) for m in config.options["modules"]]
         except:
             raise Exception("Can't find {}.type in config".format(configuration_key))
 
-        try:
-            model = getattr(module, class_name)(
-                config=config,
-                dataset=dataset,
-                configuration_key=configuration_key,
-                init_for_load_only=init_for_load_only,
-            )
-            model.to(config.get("job.device"))
-            return model
-        except ImportError:
-            # perhaps TODO: try class with specified name -> extensibility
+        for module in modules:
+            if hasattr(module, class_name):
+                model = getattr(module, class_name)(
+                    config=config,
+                    dataset=dataset,
+                    configuration_key=configuration_key,
+                    init_for_load_only=init_for_load_only,
+                )
+                model.to(config.get("job.device"))
+                return model
+        else:
```

Comment:
This code is duplicated. I suggest to factor it out to kge/misc in a method along the lines of `init(modules, class_name, *args, **kwargs)`.